main.py

```diff
@@ -692,5 +842,4 @@ def get_file_details(file_name: str) -> tuple[int, int]:
         main()
 
 
-pygame.quit()
-sys.exit()
```

Comment:
This was the monstrous line that took away so many hours of my life. Beyond demon.